challenges/database/translate.py

```diff
@@ -0,0 +1,76 @@
+import json
+import sys
+
+def main():
+  if len(sys.argv) <= 1:
+    print('Usage: ' + sys.argv[0] + ' [json file with data]')
+    sys.exit(0)
+
+  # Statement to create Users SQL table
+  print('CREATE TABLE Users (\n'
+        + '  RecordID int NOT NULL,\n'
+        + '  Name varchar(255),\n'
+        + '  CellPhone varchar(255),\n'
+        + '  WorkPhone varchar(255),\n'
+        + '  Email varchar(255),\n'
+        + '  Address varchar(255),\n'
+        + '  PRIMARY KEY (RecordID)\n'
+        + ');\n')
+  
+  # Statement to create Orders SQL table
+  print('CREATE TABLE Orders (\n'
+        + '  OrderID int NOT NULL,\n'
+        + '  BasicWidgetOrder int,\n'
+        + '  AdvancedWidgetOrder int,\n'
+        + '  ProtectionPlan boolean,\n'
+        + '  RecordID int NOT NULL,\n'
+        + '  PRIMARY KEY (OrderID),\n'
+        + '  FOREIGN KEY (RecordID) REFERENCES Users(RecordID)\n'
+        + ');\n')
+        
+  # We already have record IDs so we want to insert them ourselves
+  print('SET IDENTITY_INSERT Users ON;\n')
```

Comment:
This statement allows me to specify a specific primary key for inserting values. The default behavior for MySQL is to not provide the primary key and have it auto increment.